oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,249 @@
+Marketing Team Members, Documentation Writers, QA, and Product Managers.
+
+
+All CCs are granted some Rights to the project as part of participating - the ability to
+merge code, the final say on marketing copy, etc. In exchange for these Rights to contribute, we ask that
+CCs abide by their Responsibilities. For more on the CC roles, and Rights & Responsibilities, see:
+
+- `Core Contributor Role Definitions <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2759460357/Core+Contributor+Role+Definitions>`_
+- `Core Contributor Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2952003698/Core+Contributor+Rights+Responsibilities+excluding+code+contributors>`_
+- `Code Committer Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Rights+Responsibilities+for+Code+Contributors>`_
+
+Note that roles vary in what they need from members; the wiki will be the most up to date source
+of what each role entails and what types of commitments are expected.
+
+.. _program administration:
+
+Core Contributor Program Administration
+---------------------------------------
+
+The Core Contributor Program is administered by The Center for Re-imagining
+Learning (tCRIL). One or more persons at tCRIL (the "Program administrator(s)")
+work with the community to make sure the program runs smoothly: this entails
+facilitating communication amongst community members, `selecting new members`_,
+and ensuring that existing members renew their commitment to the community and
+abide by the Code of Conduct. Program administrators are empowered to monitor
+Core Contributor conduct, particularly whether they follow the "3 Cs"
+(Commitment, Conduct, Caliber), and may issue reminders, suspensions, and even
+removal from the Program. Program administrators can be contacted at
+``cc-program-admins@tcril.org``.
+
+The tCRIL administrator(s) are the point people for making sure that needed
+agreements are signed - this includes a Program Agreement between the CC and
+tCRIL (sent to you by a Program Administrator); for code contributors, an
+additional CLA (signed when you make your first PR); and for new entities
+joining the community, an Entity Contributor Agreement (please communicate with
+a Program Administrator about this).
+
+We are additionally asking that all employers of CCs sign the `Revised
+Declaration of Commitment to the Core Contributor Program
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3216900524>`_ to
+demonstrate that they are willing to engage with and commit resources to the
+Open edX project. We encourage a commitment of 20 hours/month from each of an
+organization's Core Contributors, however, we recognize that circumstances
+differ and a different commitment may be more appropriate. Before signing the
+agreement, please reach out to the Program Administrators to discuss.
```

Comment:
Can link to the suggested "Contact Program Administrators" subsection instead:

(I _think_ this is  [how you embed a named link to an internal anchor](https://jwodder.github.io/kbits/posts/rst-hyperlinks/#internal-hyperlink-targets), but I'm not sure.. RST is really hard!)

```suggestion
agreement, please `reach out to the Program Administrators <contact_admin_>`_ to discuss.
```